### PR TITLE
IPFM-1355: Align versions of actions with other repos

### PR DESCRIPTION
**Pull request by maksbond23**
IPFM-1355
----

Align action versions between all repositories

Details
----
* Update `actions/checkout` from `v2` to `v3`
* Update `actions/cache` from `v2.0.0` to `v3`
* Replace `richardsimko/update-tag@v1` with `scoremedia/update-tag@v2
* Replace `ncipollo/release-action@v1` with `scoremedia/release-action@v2`
* Replace `sushichop/action-repository-permission@v1` with `scoremedia/action-repository-permission@v2`
* Replace `peter-evans/create-pull-request@v3` with `scorebet/create-pull-request@v5`
`

### CODE REVIEW

File: .github/workflows/update-changelog.yml [hunk 1]
```diff
@@ -20,12 +20,12 @@ jobs:
 
     steps:
     - name: Check Acccess
-      uses: sushichop/action-repository-permission@v1
+      uses: scoremedia/action-repository-permission@v2
```
[the remark this answers is not in the document]
Comment (maksbond23) [reply .github/workflows/update-changelog.yml:23]: #cycode_ignore_insecure_pipeline_violation_here

File: .github/workflows/update-changelog.yml [hunk 2]
```diff
@@ -54,7 +54,7 @@ jobs:
         git push origin create-release --force 
     
     - name: Create PR
-      uses: peter-evans/create-pull-request@v3
+      uses: scorebet/create-pull-request@v5
```
[the remark this answers is not in the document]
Comment (maksbond23) [reply .github/workflows/update-changelog.yml:57]: #cycode_ignore_insecure_pipeline_violation_here

File: .github/workflows/create-release.yml [hunk 3]
```diff
@@ -52,7 +52,7 @@ jobs:
 
     - name: Create Release
       id: create_release
-      uses: ncipollo/release-action@v1
+      uses: scoremedia/release-action@v2
```
[the remark this answers is not in the document]
Comment (maksbond23) [reply .github/workflows/create-release.yml:55]: #cycode_ignore_insecure_pipeline_violation_here

File: .github/workflows/create-release.yml [hunk 4]
```diff
@@ -23,7 +23,7 @@ jobs:
         echo "::set-output name=tag::$tag"
 
     - name: Update tag
-      uses: richardsimko/update-tag@v1
+      uses: scoremedia/update-tag@v2
```
[the remark this answers is not in the document]
Comment (maksbond23) [reply .github/workflows/create-release.yml:26]: #cycode_ignore_insecure_pipeline_violation_here

**Review by canvckle — APPROVED** [review]

**Review by tahirmt — APPROVED** [review]

**Pull request merged**